user/TASK/gimbal_task/gimbal_task.c

```diff
@@ -89,29 +111,58 @@ static void initialization(Gimbal_t *gimbal_ptr){
     gimbal_ptr->launcher = get_launcher_pointer();
     fp32 pid_constants_yaw[3] = {pid_kp_yaw, pid_ki_yaw, pid_kd_yaw};
     fp32 pid_constants_pitch[3] = {pid_kp_pitch, pid_ki_pitch, pid_kd_pitch};
-
+    
+    gimbal_ptr->yaw_setpoint[0] = 0.0;
+    gimbal_ptr->yaw_setpoint[1] = -1.0;
+    gimbal_ptr->yaw_position[0] = 0.0;
+    gimbal_ptr->yaw_position[1] = -1.0;
+    gimbal_ptr->yaw_error = 0.0;
+    
     PID_Init(&(gimbal_ptr->pitch_motor.pid_controller), PID_POSITION, pid_constants_pitch, max_out_pitch, max_i_term_out_pitch);
     PID_Init(&(gimbal_ptr->yaw_motor.pid_controller), PID_POSITION, pid_constants_yaw, max_out_yaw, max_i_term_out_yaw);
     
     gimbal_ptr->rc_update = get_remote_control_point();
     
-    gimbal_ptr->pitch_motor.pos_set = 5500;
+    gimbal_ptr->pitch_motor.pos_set = 3000;
     gimbal_ptr->yaw_motor.pos_set = 6000;
```

Comment:
Is this dependent on how the motor is physically mounted? - If it is, 3000 and 6000 should be defined in the header file

Reply:
Actually, they should be defined regardless

Reply:
Added constant